zipline/pipeline/engine.py

```diff
@@ -62,6 +65,37 @@ def run_pipeline(self, pipeline, start_date, end_date):
         """
         raise NotImplementedError("run_pipeline")
 
+    @abstractmethod
+    def run_chunked_pipeline(self, pipeline, start_date, end_date, chunksize):
+        """
+        Compute values for `pipeline` for date range chunks the size
+        `chunksize`. Computing in chunks could reduce the amount of memory
```

Comment:
Is there something more specific we can say here besides 'could reduce the amount of memory'?